apps/stripe/src/modules/app-config/stripe-config.ts

```diff
@@ -0,0 +1,63 @@
+import { err, ok } from "neverthrow";
+
+import { StripePublishableKey } from "@/modules/stripe/stripe-publishable-key";
+import { StripeRestrictedKey } from "@/modules/stripe/stripe-restricted-key";
+
+export class StripeConfig {
+  private constructor(
+    private deps: {
+      name: string;
+      id: string;
+      restrictedKey: StripeRestrictedKey;
+      publishableKey: StripePublishableKey;
+    },
+  ) {}
+
+  static createFromPersistedData(args: {
```

Comment:
this class doesn't really "know" if data were persisted or not, and shouldn't case. its just `create()` most likely